### PR TITLE
Add coverage script in the admin

### PR DESCRIPTION
**Pull request by simotae14**
### What does it do?

I have added a script to collect the coverage of the front end unit tests inside the admin

### Why is it needed?

To know the percentage of the code covered with the tests

### How to test it?

Just run 
`
yarn test:front:ce:cov
`
from the packages/core/admin folder

### CODE REVIEW

**Review by gu-stav — DISMISSED** [review]
I'm fine with that and think it is a good addition, but would also like to hear e.g. @alexandrebodin s opinion.

File: packages/core/admin/package.json [hunk 1]
```diff
@@ -34,7 +34,8 @@
     "test:front": "cross-env IS_EE=true jest --config ./jest.config.front.js",
     "test:front:watch": "cross-env IS_EE=true jest --config ./jest.config.front.js --watchAll",
     "test:front:ce": "cross-env IS_EE=false jest --config ./jest.config.front.js",
-    "test:front:watch:ce": "cross-env IS_EE=false jest --config ./jest.config.front.js --watchAll"
+    "test:front:watch:ce": "cross-env IS_EE=false jest --config ./jest.config.front.js --watchAll",
+    "test:front:ce:cov": "cross-env IS_EE=false jest --config ./jest.config.front.js --coverage --collectCoverageFrom='<rootDir>/packages/core/admin/admin/**/*.js' --coverageDirectory='<rootDir>/packages/core/admin/coverage'"
```
Comment (alexandrebodin) [inline packages/core/admin/package.json:38]: Any reason not to use the jest config file to set the convert config instead of using the scripts ?

**Review by alexandrebodin — CHANGES_REQUESTED** [review]

File: packages/core/admin/package.json (hunk 1 above)
Comment (simotae14) [reply packages/core/admin/package.json:38]: nope, I can try with the jest config file

Comment (simotae14) [reply packages/core/admin/package.json]: please, let me know if it is ok now, thanks

**Review by alexandrebodin — APPROVED** [review]

File: packages/core/admin/package.json (hunk 1 above)
Comment (alexandrebodin) [reply packages/core/admin/package.json]: Looks good yes

**Review by gu-stav — CHANGES_REQUESTED** [review]
@simotae14 Could you also update the `test:front:cov` script in the helper-plugin please, so that they are consistent?

**Review by alexandrebodin — APPROVED** [review]

**Review by gu-stav — APPROVED** [review]
Great addition! Thank you.

**Pull request merged**